control/action.yaml

```diff
@@ -0,0 +1,44 @@
+openapi: 3.1.0
+
+info:
+  title: Action models
```

Comment:
Lets not use action models - lets start this off as DeviceState and with this lacp state as the first choice.  This can then be expanded in subsequent PRs.

Reply:
done.